packages/xs-extra/xenctrl.0.1.0/opam

```diff
@@ -24,5 +24,5 @@ depexts: [
 
 dev-repo: "git+https://github.com/xapi-project/xenctrl.git"
 url {
-  src: "https://github.com/xapi-project/xenctrl/archive/master.tar.gz"
+  src: "https://github.com/xapi-project/xenctrl/archive/v0.1.0.tar.gz"
```

Comment:
The commit has two tags: v1.1.0 and v0.1.0, which is confusing

Reply:
this has been remedied